gamestonk_terminal/fred/README.md

```diff
@@ -0,0 +1,46 @@
+# fred
+
+The purpose of fred is to have the ability to look at economic data.
+
+* [GDP](#GDP)
+    * Get GDP data
+* [cust](#cust)
+    * Get custom fred data  
+
+## GDP <a name="GDP"></a>
+```text
+usage: GDP [-n N_TO_GET] [-s START_DATE] [-p PLOT]]
+```
+Gets GDP data.  Data is released quarterly
+* -n : Number of points to display.  If unspecified, it will display all available.
+* -s : Start Date.  Format m/d/y. First point to get
+* -p : Option to plot.  Defaults to False
```

Comment:
Is it enough to only look at the data? Genuinely asking, as I've not used this data before. 

If the answer to this is no, I'd make plot as default. Otherwise, this is good.

Reply:
So I can't say I've used it either.  My instinct would be that it depends on the data.  For example, since GDP is quarterly, doesn't make sense to plot if I am trying to YOLO based on GDP growth from last week.

If I was looking at unemployment data, probably better to plot.  Ill default to true for now.

Reply:
Sounds good to me!